src/minefield/setup_game.tests.cpp

```diff
@@ -1,28 +1,23 @@
 #include <gtest/gtest.h>
 
-#include <minefield/create_test_game.h>
+#include <minefield/testing_utils.h>
 #include <minefield/minefield.h>
 #include <minefield/setup_game.h>
 #include <minefield/show_board.h>
 
+auto getOutputofReading = [](unsigned int min, unsigned int max, unsigned int value) { // gets the console output when an input string is sent to the readIntInRange function
```

Comment:
#should be a function, not a lambda

#should follow naming convention, missing capital O at `of`